admin/class-yoast-form.php

```diff
@@ -93,6 +93,21 @@ public function set_option( $option_name ) {
 	}
 
 	/**
+	 * Sets a value in the options.
```

Comment:
Please add the default behaviour for overriding in the description: "if it is not already set."